pkgs/development/ocaml-modules/phylogenetics/default.nix

```diff
@@ -14,6 +15,9 @@
 , printbox-text
 }:
 
+lib.throwIf (lib.versionAtLeast ocaml.version "5.0")
```

Comment:
Not tested but I think 0.2 version works with ocaml 5 `https://github.com/biocaml/phylogenetics/releases/download/v0.2.0/phylogenetics-0.2.0.tbz`
cc @anmonteiro